src/OpenLoco/src/Ui/Windows/ScenarioOptions.cpp

```diff
@@ -1222,17 +1222,17 @@ namespace OpenLoco::Ui::Windows::ScenarioOptions
             {
                 case widx::change_name_btn:
                 {
-                    strncpy(S5::getOptions().scenarioName, input, sizeof(S5::Options::scenarioName) - 1);
-                    S5::getOptions().scenarioName[sizeof(S5::Options::scenarioName) - 1] = '\0';
+                    strncpy(getOptions().scenarioName, input, sizeof(OpenLoco::Options::scenarioName) - 1);
+                    getOptions().scenarioName[sizeof(OpenLoco::Options::scenarioName) - 1] = '\0';
                     unloadScenarioTextObjects();
                     self.invalidate();
                     break;
                 }
 
                 case widx::change_details_btn:
                 {
-                    strncpy(S5::getOptions().scenarioDetails, input, sizeof(S5::Options::scenarioDetails) - 1);
-                    S5::getOptions().scenarioDetails[sizeof(S5::Options::scenarioDetails) - 1] = '\0';
+                    strncpy(getOptions().scenarioDetails, input, sizeof(OpenLoco::Options::scenarioDetails) - 1);
+                    getOptions().scenarioDetails[sizeof(OpenLoco::Options::scenarioDetails) - 1] = '\0';
```

Comment:
Any particular reason for these extra qualifiers? Clashes with ui/window namespaces?